.github/workflows/build-test-deploy.yml

```diff
@@ -26,6 +26,9 @@ jobs:
         uses: ./.github/actions/build
         with:
           NEXT_PUBLIC_ENV: 'prod'
+          NEXT_PUBLIC_TRANSAK_APP_URL: 'https://global.transak.com'
+          NEXT_PUBLIC_TRANSAK_API_URL: 'https://api.transak.com'
+          NEXT_PUBLIC_TRANSAK_API_KEY: '799087ea-c2be-44b8-a0fe-56d96b37ee3c'
```

Comment:
I'm not sure these are totally necessary in this spot.  @defispartan or @RoRu can confirm though

Reply:
This block is saying "use everything in `/build/actions.yml` with these additional variables.  Since they're already added in the original build file, they're not needed here.

Reply:
I'm 99% certain they are redundant here, and can be removed.

Reply:
Can confirm, these won't do anything here, better to remove them

Reply:
Right, it's been a minute, but these can be removed.  If adding in this way, then `actions.yml` would need to take in the dynamic values, and they'd be defined out of scope at this file.  This isn't what we want, so please remove these.

Reply:
Done

Reply:
Great, thanks!